policy/modules/services/certbot.fc

```diff
@@ -2,3 +2,9 @@
 /usr/bin/letsencrypt	--	gen_context(system_u:object_r:certbot_exec_t,s0)
 /var/lib/letsencrypt(/.*)?	gen_context(system_u:object_r:certbot_lib_t,s0)
 /var/log/letsencrypt(/.*)?	gen_context(system_u:object_r:certbot_log_t,s0)
+
+/etc/acme-sh(/.*)?		gen_context(system_u:object_r:certbot_lib_t,s0)
+/usr/share/acme\.sh(/.*)?		gen_context(system_u:object_r:certbot_lib_t,s0)
```

Comment:
I strongly suspect that this entry should not be needed.

Reply:
Ok. I might need to adapt this to more of a general use case scenario. This is more or less translated directly from how I have it set up on my boxes. Perhaps just remove the context for `/usr/share/acme.sh` and move acme.sh's lib dir to `/var/lib/acme-sh` for consistency?

Reply:
I took another look at this. On my system acme.sh stores its additional scripts for deployment and such here in `/usr/share/acme-sh/{deploy,dnsapi,notify}`, so these almost certainly should be `certbot_exec_t`.